react/utils/client/fetch.ts

```diff
@@ -0,0 +1,21 @@
+import {format, parse} from 'url'
+
+export const IOFetch = async (uri: string | Request, init?: RequestInit): Promise<Response> => {
+  const {useGetInBatch = false, body} = init || {} as any
+  const formattedInit = {...init}
+  let formattedUri = uri
+
+  if (useGetInBatch && typeof uri === 'string' && formattedInit) {
+    const parsedUri = parse(uri, true)
+
+    delete parsedUri.search
+    parsedUri.query = {...parsedUri.query, batch: body}
```

Comment:
Let's make this into a `batch[]={}&batch[]={}&...` so we use rails' and PHP format.